requirements/default.txt

```diff
@@ -196,8 +196,9 @@ importlib==1.0.3 \
     --hash=sha256:65f342a604a2e1028707c5e055266ab2431c26e20fe10780b423320870884dac
 
 # Improved template engine
-Jinja2==2.7.3 \
-    --hash=sha256:2e24ac5d004db5714976a04ac0e80c6df6e47e98c354cb2c0d82f8879d4f8fdb
+Jinja2==2.8 \
+    --hash=sha256:1cc03ef32b64be19e0a5b54578dd790906a34943fe9102cfdae0d4495bd536b4 \
+    --hash=sha256:bc1ff2ff88dbfacefde4ddde471d1417d3b304e8df103a7a9437d47269201bf4
```

Comment:
I found this discussion, which I think is the source of the "we don't use wheels" rule for kitsune and input: https://github.com/mozilla/kitsune/pull/2482

We never used peep on MDN, preferring submodules instead 😱. With pip 8, hashes were added, and we switched from submodules to requirement files. We also use a constraints file, so anything less than pip 7.1 will fail, and we use pip 8 in the deployment pipeline.  With [hashin](https://github.com/peterbe/hashin), it's easier to include all the hashes than exclude any, so we'll keep both if available. I think kitsune should update to pip 8 and require it as well, but that's a different project.